app/src/pages/inside/profilePage/apiKeys/apiKeysBlock/apiKeysBlock.jsx

```diff
@@ -56,13 +59,42 @@ const messages = defineMessages({
 export const ApiKeysBlock = ({ apiKeys }) => {
   const { formatMessage } = useIntl();
   const dispatch = useDispatch();
+  const { trackEvent } = useTracking();
 
-  const onGenerateClick = () => dispatch(showModalAction({ id: 'generateApiKeyModal' }));
-  const onRevokeClick = (key) => dispatch(showModalAction({ id: 'revokeApiKeyModal', data: key }));
+  const onGenerateClick = () => {
+    dispatch(showModalAction({ id: 'generateApiKeyModal' }));
+    trackEvent(PROFILE_EVENTS.CLICK_GENERATE_API_KEY_BUTTON(true));
+  };
+
+  const onRevokeClick = (key) => {
+    dispatch(showModalAction({ id: 'revokeApiKeyModal', data: key }));
+    trackEvent(PROFILE_EVENTS.CLICK_REVOKE_BUTTON);
+  };
+
+  const onDocumentationClick = (event) => {
+    const { tagName } = event.target;
+
+    if (tagName === 'A') {
+      trackEvent(PROFILE_EVENTS.CLICK_DOCUMENTATION_LINK(true));
```

Comment:
Looks like this violates one of the clean code principles.
If we are using static bool arg, it is better to create two different functions

Reply:
Done.